app/articles.models.js

```diff
@@ -13,15 +13,21 @@ exports.retriveArticleById = (id) => {
 	});
 };
 
-exports.retriveArticles = (sort_by = 'created_at', order = 'desc') => {
+exports.retriveArticles = (topic, sort_by = 'created_at', order = 'desc') => {
 	let results;
+	let queryValues = [];
 	let queryString = `select * from articles `;
 
+	if (topic) {
+		queryValues.push(topic);
+		queryString += `where topic = $1 `;
+	}
+
 	if (sort_by) {
```

Comment:
The sprint hasn't asked you to add sort_by or order in to the query yet, I think this is one of the advanced tasks.